lib/Install.js

```diff
@@ -19,18 +24,8 @@ function install(
     // Recreate the directory to remove any artifacts from the last time
     // someone ran `elm-test install`. We do not delete this directory after
     // the installation finishes in case the user needs to debug the test run.
-    // We can replace this with just `fs.rmSync(installationScratchDir, { recursive: true, force: true })`
-    // when Node.js 12 is EOL 2022-04-30 and support for Node.js 12 is dropped.
-    // `fs.rmSync` was added in Node.js 14.14.0, which is also when the
-    // `recursive` option of `fs.rmdirSync` was deprecated. The `if` avoids
-    // printing a deprecation message.
     // $FlowFixMe[prop-missing]: Flow does not know of `fs.rmSync` yet.
-    if (fs.rmSync !== undefined) {
-      fs.rmSync(installationScratchDir, { recursive: true, force: true });
-    } else if (fs.existsSync(installationScratchDir)) {
-      // $FlowFixMe[extra-arg]: Flow does not know of the options argument yet.
-      fs.rmdirSync(installationScratchDir, { recursive: true });
-    }
```

Comment:
As I said in an earlier comment, it was decided to support Node.js 12 for the time being, so this needs to be reverted so CI can pass.

It’s fine though if the new feature is Node.js 14+ (as long as CI passes).

Reply:
Reverted in 0d5d3c2